operator/pkg/metrics/monitoring.go

```diff
@@ -0,0 +1,237 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package metrics defines metrics and monitoring functionality
+// used throughout operator.
+package metrics
+
+import (
+	"istio.io/pkg/monitoring"
+)
+
+// OperatorVersionLabel describes version of running binary.
+var OperatorVersionLabel = monitoring.MustCreateLabel("version")
+
+// MergeErrorLabel describes the type of merge error.
+var MergeErrorLabel = monitoring.MustCreateLabel("error_type")
+
+// RenderErrorLabel describes the type of the error while rendering.
+var RenderErrorLabel = monitoring.MustCreateLabel("render_error")
+
+var (
```

Comment:
nit: any reason why some vars are declared in a block and others separately ?

Reply:
Initially I thought of grouping by their areas. When I look at it now, it didn't make much sense. So I put all of that in one group - like I have done for metrics.